src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/bitbucket/server/ServerApiClient.java

```diff
@@ -64,7 +64,21 @@ public boolean hasBuildStatus(String owner, String repositoryName, String revisi
 
     @Override
     public void setBuildStatus(String owner, String repositoryName, String revision, BuildState state, String buildUrl, String comment, String keyEx) {
-        setBuildStatus(state, buildUrl, comment, keyEx, buildStateV1(revision));
+        String url = buildStateV1(revision);
+        String computedKey = computeAPIKey(keyEx);
+        ServerPullrequest.CommitBuildState commit = new ServerPullrequest.CommitBuildState();
+
+        // FIXME TODO WTF? Where do you put the comment?
```

Comment:
For the server api v1 I found no place to include the comment, in the v2 I know its possible

Reply:
Lol! I was wondering! I'll leave the comment (if it's ok with you), so that we remember to add it in, if server ever goes to v2.

Reply:
Not sure if you are aware but Bitbucket cloud (what we use :-) is deprecating all v1 APIs in April 2019.  So everything in v1 _should_ have a v2 replacement

https://developer.atlassian.com/cloud/bitbucket/deprecation-notice-v1-apis/

Reply:
@stevemuskiewicz : While I agree with you 100% . . . 

The ServerApiClient is the Bitbucket SERVER side of things, not the cloud.

Our cloud api is 100% v2.

There is exactly one user, that I know of, using bitbucket server, and he is the approver of this fix. Since I have no way to test his particular v1 endpoints on his Bitbucket Server, I'm planning on merging . .

Reply:
@CodeMonk gotcha, makes sense.  Thanks! 😀